barman/cli.py

```diff
@@ -98,16 +254,13 @@ def list_servers(minimal=False):
     output.close_and_exit()
 
 
-@named("list-server")
-@wraps(list_servers)
-@arg("--minimal", help="machine readable output")
-def list_server(minimal=False):
-    return list_servers(minimal)
-
-
-@arg(
-    "--keep-descriptors",
-    help="Keep the stdout and the stderr streams attached to Barman subprocesses.",
+@command(
+    [
+        argument(
+            "--keep-descriptors",
+            help="Keep the stdout and the stderr streams attached to Barman subprocesses",
+        )
+    ]
 )
 def cron(keep_descriptors=False):
```

Comment:
This is another place we were relying on argh magic which I missed in the last review - we need to do the same change we did for `list-servers`, specifically: accept `args` instead of `keep_descriptors`, add `action="store_true"` the `--keep-descriptors` argument, and refer to `args.keep_descriptors` inside the `cron` function.